README.md

```diff
@@ -20,7 +20,7 @@ http://pcottle.github.com/learnGitBranching/
 By default the application launches in "sandbox mode" with a basic repository already created. Here you can enter commands and mess around with a repository as much as you like. Keep in mind you can
 
 * `undo` to undo the effects of the last command
-* `reset` to start over from a clean slate (works in levels too)
+* `reset` to start over with a clean slate (works in levels too)
```

Comment:
I don't agree that this line needs changing. "Start from" (and by extension "start over from") is perfectly valid English AFAIK.